source/calibration.rst

```diff
@@ -116,3 +116,95 @@ back to the previous calibration with:
 
     >$ calibrate_robot --restore
 
+
+Calibrating Fetch Torso
+-----------------------
+When the torso controller board is first powered, it uses measurements from two different
+position sensors to determine the absolute starting position of the torso.
+Once the absolute starting position of the torso is determined, the position measurement
+will retain millimeter precision as long torso remains powered.
+
+To work properly, the two torso sensors must be calibrated together.
+If the sensors are not properly calibrated, the calculation of the initial torso position
+could be incorrect in some situations. This problem is more likely to occur if the torso is
+first powered when in the "up" position.
+A bad torso position should be easy to detect since the reported torso position will
+always be off by more than +/-10cm.
```

Comment:
probalby should add "when viewed in RVIZ" or something like this -- so that users understand how to see the error